source/video_core/src/video_core/vulkan/renderer.cpp

```diff
@@ -1452,14 +1452,14 @@ bool Renderer::BlitImage(Context& context, uint32_t /* TODO: PAddr */ input_addr
                               ImageLayoutTransitionPoint::ToTransferDstGeneral());
 
         auto full_image = vk::ImageSubresourceLayers { vk::ImageAspectFlagBits::eColor, 0, 0, 1 };
-        const auto input_start_offset = vk::Offset3D { 0, static_cast<int32_t>(input_start_y), 0 };
+        const auto input_start_offset = vk::Offset3D { 0, static_cast<int32_t>(flip_y ? input_end_y : input_start_y), 0 };
         const auto output_start_offset = vk::Offset3D { 0, 0, 0 };
-        const auto input_end_offset = vk::Offset3D { static_cast<int32_t>(input_width), static_cast<int32_t>(input_end_y), 1 };
+        const auto input_end_offset = vk::Offset3D { static_cast<int32_t>(input_width), static_cast<int32_t>(flip_y ? input_start_y : input_end_y), 1 };
         const auto output_end_offset = vk::Offset3D { static_cast<int32_t>(output_width), static_cast<int32_t>(output_height), 1 };
         {
             // Validate that in each dimension, we're either copying 1:1 or scaling 2:1
             const auto copied_input_width = input_end_offset.x - input_start_offset.x;
-            const auto copied_input_height = input_end_offset.y - input_start_offset.y;
+            const auto copied_input_height = flip_y ? (input_start_offset.y - input_end_offset.y) : (input_end_offset.y - input_start_offset.y);
```

Comment:
Unconditionally taking the absolute value here is a bit simpler to follow:
```suggestion
            const auto copied_input_height = std::abs(input_start_offset.y - input_end_offset.y);
```